### PR TITLE
Enabled additional rules for SwiftFormat

### PR DESCRIPTION
**Pull request by eviltofu**
Added the rules mentioned in https://github.com/AlphaWallet/alpha-wallet-ios/issues/6583#issuecomment-1521104075

### CODE REVIEW

File: .swiftformat [hunk 1]
```diff
@@ -7,87 +7,97 @@
...
 #Files
 
-#--exclude build,vendor
-#--exclude Pods,Generated
+--exclude build,vendor
+--exclude Pods,Generated
 #Excluded because of error while processing
 #--exclude AlphaWallet/Browser/ViewModel/Dapps.swift
+
+# I added these
```
Comment (hboon) [inline .swiftformat:95]: Remember to remove the comment and move the rules up, to before `#Files`

**Review by hboon — COMMENTED** [review]
@eviltofu remember to remove the `#TODO enable` comments too

Seems OK to me. I ran it on `modules/` but gave up check after going through half of it.

`trailingCommas` is a little strange, but I can live with it.

@oa-s would you give this a quick pass to see if there's anything glaring? You can run it on a subset and look at diffs:

```
pods/SwiftFormat/CommandLineTool/swiftformat --config .swiftformat  modules/AlphaWalletOpenSea
```

**Comment by hboon** [comment]
BTW, @eviltofu can you help add a TODO inside `resources/SwiftFormatFiles.xcfilelist` to ignore files such as `modules/AlphaWalletFoundation/AlphaWalletFoundation/NFT/Enjin/API.swift` that are autogenerated? Maybe `R.generated` too

**Comment by eviltofu** [comment]
> BTW, @eviltofu can you help add a TODO inside `resources/SwiftFormatFiles.xcfilelist` to ignore files such as `modules/AlphaWalletFoundation/AlphaWalletFoundation/NFT/Enjin/API.swift` that are autogenerated? Maybe `R.generated` too

This should be in the config file.

**Comment by eviltofu** [comment]
I get 
```
error: Unexpected end of file at 121:1 in /Users/jerome/Desktop/work/alpha-wallet-ios/AlphaWallet/Browser/ViewModel/Dapps.swift.
```
But this file is not in the AlphaWallet project. Can I remove this?

**Comment by hboon** [comment]
Exclude the file from SwiftFormat, but keep in repo. We are still maintaining it; just not including it

**Comment by oa-s** [comment]
> would you give this a quick pass to see if there's anything glaring? You can run it on a subset and look at diffs:

looks good, a lot of changes in Foundation and Web3 pods, but looks good

File: .swiftformat [hunk 2]
```diff
@@ -6,88 +6,83 @@
...
+--enable duplicateImports
+--enable consecutiveBlankLines
+--enable consecutiveSpaces
+
 #Files
 
-#--exclude build,vendor
-#--exclude Pods,Generated
-#Excluded because of error while processing
-#--exclude AlphaWallet/Browser/ViewModel/Dapps.swift
```
Comment (hboon) [inline .swiftformat:93]: We need this one back, uncommented, along with the comment. Then should be good to merge

**Review by hboon — APPROVED** [review]

**Pull request merged**